bench-vortex/src/bin/clickbench.rs

```diff
@@ -455,87 +402,3 @@ fn execute_queries(
 
     query_measurements
 }
-
-/// Executes a single ClickBench query using DataFusion.
-///
-/// # Returns
-///
-/// - The duration of the fastest execution
-/// - The execution plan used for the query
-#[allow(clippy::unwrap_used)]
-fn benchmark_datafusion_query(
```

Comment:
All benchmarks now call `benchmark_datafusion_query` & `benchmark_duckdb_query` from the `engines` module.